web_advanced_search/static/src/js/control_panel/advanced_select_list_controller.js

```diff
@@ -1,11 +1,14 @@
 /* Copyright 2021 Therp BV {https://therp.nl>.
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
  */
+
 odoo.define("web_advanced_search.select_create_list_controller", function (require) {
     "use strict";
-    const controller_registry = require("web.select_create_controllers_registry");
-    const selectCreateController = controller_registry.SelectCreateListController;
-
+    var view_dialog = require("web.view_dialogs");
```

Comment:
Why a var instead of a const?

Reply:
it could be a const. yes. changing.

Reply:
I see a lot of var stil:
```
    var view_dialog = require("web.view_dialogs");
    var controller_registry = require("web.select_create_controllers_registry");
    require("web._select_create_controllers_registry");
    var selectCreateController = controller_registry.SelectCreateListController;
```
Is that a quirk in github, or are these still var?

Reply:
changed no variable at all was needed, just  require("web.view_dialogs") is enough

        ```
        require("web.view_dialogs");                                                        
         var controller_registry = require("web.select_create_controllers_registry");        
         require("web._select_create_controllers_registry");                                 
         var selectCreateController = controller_registry.SelectCreateListController;   
```
controller registry and controller_registry.SelectCreateListController are vars because i change them.

this is what i see in code, but i have noticed wierdness in github too.. I just rebased

Reply:
@NL66278 just rebased let me know if you see it normal. i see it normal now.

Reply:
You only need var when you re-assign the variable later. Calling include on the object referenced by a const is not considered a change as the reference itself if not touched.